pkgs/development/python-modules/pyvips/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildPythonPackage, fetchPypi, python, pytestrunner, gcc, glib, cffi
+, python3Packages, pkgconfig, stdenv, vips, lib}:
+
+buildPythonPackage rec {
+
+  pname = "pyvips";
+
+  version = "2.1.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pg0dxhxgi2m7bb5bi5wpx9hgnbi8ws1bz6w2dldbhi52pizghl4";
+  };
+
+  buildInputs = [
+    python3Packages.pytestrunner
+    python3Packages.pkgconfig
+  ];
+
+  propagatedBuildInputs = [
+    python3Packages.cffi
+  ];
+
+  doCheck = false;
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.